### ISSUE TITLE
governance: Using stalebot for Spoon 

### ISSUE THREAD

**Issue by surli** [INRIA/spoon#2556]
I propose that we use a Github bot to automatically close PR/issues after a period of inactivity. 

I've seen lot of big projets using that kind of bot (such as Travis, Docker, ...): I was mixed about it at the beginning, but since @monperrus enforce the closing of PR/issues manually to maintain a "clean" repository, I really think it might be useful to avoid doing ourselves this boring task.

An example of such as bot is available there: https://probot.github.io/apps/stale/

**Comment by monperrus** [comment]
I like your idea.

Is there a free SaaS service for this?

Otherwise, I propose to host it at OW2.

**Comment by surli** [comment]
I don't think you need to host anything. AFAIK you install it as a Github app in the repo and you need to configure it by adding a file in the repo, like for the travis.yml.
